package.json

```diff
@@ -87,6 +87,7 @@
     "q": "1.0.1",
     "qrcode.react": "0.6.1",
     "qs": "4.0.0",
+    "ramda": "0.22.1",
```

Comment:
>This didn't seem like a problem to me since we're removing hundreds of kilobytes by taking out the iframe'd code.

This is nice, sure, but is not relevant to a discussion on whether it makes sense to include a new package. For all intents and purposes, `pick` is the same whether it's from Ramda or from Lodash, so importing both is a waste of bytes as far as I see it.

Noting that the pull request is still in progress, I'll wait and let others weigh in with their own thoughts. That said, I have strong reservations about this change since I feel that for the sake of maintainability, we should be consensus-seeking in our approaches to common problems.